internal/filterapi/runtimefc/runtimefc.go

```diff
@@ -0,0 +1,84 @@
+// Copyright Envoy AI Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+// Package runtimefc is the API for the construct of runtime filter configurations parsed from filterapi.Config.
+//
+// Decoupled from filterapi to avoid circular dependencies.
```

Comment:
This is the reason why we cannot have this directly inside the filterapi package